OneCommunityGlobal/HGNRest
# Carlos add hours by team member visualization project reports

## Related PRS (if any):
This Backend  PR is related to supporting proposal changes in Frontend PR Carlos

# Description
The graph currently displays weekly committed hours by project member. However, it does not show the actual hours worked by member on each project.
 
(PRIORITY HIGH) Jae: Add “Hours by Team Member” Visualization to the Projects Reports page  (WIP Carlos)

## Main changes explained:
- in  getTimeEntriesForSpecifiedProjectadd,  add firstName lastName  and isActive properties in 'TimeEntry' Schema from 'userProfile' Schema with will be received in the fronted.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as Owner, manager or admin user
5. Folllow Frontend intructions

## Requested job:

![task](https://github.com/OneCommunityGlobal/HGNRest/assets/55204862/34abae75-c286-464a-af41-59f101b2f667)

## video before the change:
https://www.loom.com/share/7be14c4aec464c5fbb9669a11c4d63b5?sid=586d7ba2-8f94-4a4a-99e6-6725b31d4236

## video after the change:
https://www.loom.com/share/889ae08438b94daca7d4f63f9905cb07?sid=2039627e-1faa-43ae-84a6-6d5f3b8a53af


## Note:
1.  You must add a project for at least one user ( set as a  poject member)
2. You must log hours for this member(s) in this project.
3. Go to dashboard => report => type the project  => PieChart  => pick the charts
4.See videos above for more info
